spec/factories/users.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :random_user, class: User do
+    email { Faker::Internet.email }
+    password { Faker::Internet.password }
+
+    trait :normal do
+      role { Role.find_by(name: 'User') }
+      # association :role, factory: :role
```

Comment:
don't keep unwanted code unless it might be required in the next few commits or branches.